GoogleDataTransport/GDTCORLibrary/GDTCORPlatform.m

```diff
@@ -137,7 +137,9 @@ GDTCORNetworkMobileSubtype GDTCORNetworkMobileSubTypeMessage() {
       networkCurrentRadioAccessTechnology = networkCurrentRadioAccessTechnologyDict.allValues[0];
     }
   } else {
+#if defined(__IPHONE_8_0) && __IPHONE_OS_VERSION_MIN_REQUIRED < 120000
```

Comment:
Is the `defined(__IPHONE_8_0)` required?

Reply:
I don't think so, since it should always be defined for the Xcode versions we support.

Reply:
It limits the check to iOS. But you are right, `TARGET_OS_IOS` would be a better condition. Though probably it can be removed at all since everything is wrapped into `#if TARGET_OS_IOS` already.

Reply:
I've left `TARGET_OS_IOS` in the hope to help avoid extra confusion when adding support for other platforms with different OS versioning (e.g. watchOS).